cmd/web.go

```diff
@@ -580,6 +580,11 @@ func runWeb(ctx *cli.Context) error {
 			}, reqSignIn, reqRepoWriter)
 		}, repo.MustEnableWiki, context.RepoRef())
 
+		m.Group("/wiki", func() {
```

Comment:
Related closely. To support all types of embedding images into markdown files (both in wiki and repo itself) I changed wiki content loading scheme. Without this change markdown files with HTML <img> tags inside would fail to load pictures.